### PR TITLE
Creating a file with things to do in my hometown - Birzebbuga, Malta

### LINKED ISSUE #73: Create KLia hometown file
## Birżebbuġa, Malta
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [x] Have a conversation
- [x] Get a :+1: 
- [x] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by KLia**
it's not done yet though
resolves #73

### CODE REVIEW

File: birzebbuga.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Birzebbuga, Malta
+
+### Places to eat
```
Comment (KLia) [inline birzebbuga.md:3]: @abcfantasy have you been here?

File: birzebbuga.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Birzebbuga, Malta
```
Comment (KLia) [inline birzebbuga.md:1]: @redfreakly you should visit

File: birzebbuga.md (hunk 1 above)
Comment (abcfantasy) [inline birzebbuga.md:3]: X'hemm x'tara? :laughing:

File: birzebbuga.md (hunk 2 above)
Comment (mledson) [inline birzebbuga.md:1]: @KLia Would love to visit Malta someday! When is a good time of the year to visit?

File: birzebbuga.md (hunk 1 above)
Comment (KLia) [inline birzebbuga.md:3]: :(

File: birzebbuga.md (hunk 2 above)
Comment (KLia) [inline birzebbuga.md:1]: @redfreakly any time of the year ;)

File: birzebbuga.md (hunk 1 above)
Comment (abcfantasy) [inline birzebbuga.md:3]: Aw aw, niccajtaw ta ahna, niccajtaw :stuck_out_tongue:

**Comment by KLia** [comment]
This looks :+1: :+1: :+1: :+1: :+1: 
despite what @abcfantasy says

**Comment by abcfantasy** [comment]
This should be merged. Birzebbuga is so cool

**Pull request merged**